src/client/pages/CheckoutDetails/components/Group.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+
+const { gray300 } = colorsV3
+
+export const Group = styled.div`
```

Comment:
Perhaps a little too generic name => what is it a group of?

Reply:
I couldn't think of a better name for now, do you have any suggestions or is it okay if I leave it? :)